### PR TITLE
feat(documentation): add main storybook

### PR DESCRIPTION
**Pull request by gfellerph**

### CODE REVIEW

File: packages/documentation/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials"
+  ],
+  "framework": "@storybook/web-components",
+  "refs": {
+    styles: {
+      title: "Basics",
+      url: "http://localhost:6007"
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/.storybook/main.js]: Updated according to https://storybook.js.org/docs/react/sharing/storybook-composition#compose-storybooks-per-environment

File: packages/documentation/.storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    components: {
+      title: "Components",
+      url: "http://localhost:6008"
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/documentation/.storybook/main.js]: Updated according to https://storybook.js.org/docs/react/sharing/storybook-composition#compose-storybooks-per-environment

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "swisspost-design-system",
+	"name": "@swisspost/design-system",
   "description": "The Swiss Post Design System pattern library for a consistent and accessible user experience across the web platform.",
-  "author": "Swiss Post <oss@post.ch>",
+	"author": "Swiss Post <oss@post.ch>",
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply package.json]: Done

File: .github/workflows/release.yaml [hunk 4]
```diff
@@ -51,3 +51,9 @@ jobs:
         run: |
           netlify link --id $NETLIFY_SITE_ID
           netlify deploy --build false --dir packages/demo/dist/demo --prod
+
+      # Publish storybook
+      - name: Publish storybooks
+        if: steps.cahngesets.outputs.published == 'true'
+        id: storybook
+        run: pnpm --filter design-system-documentation run chromatic --project-token=${{ secrets.STORYBOOK_TOKEN }}
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply .github/workflows/release.yaml:59]: Done

File: package.json [hunk 5]
```diff
@@ -14,10 +14,11 @@
...
     "bootstrap": "pnpm install && pnpm -r build",
     "start": "pnpm demo:start",
-    "demo:start": "pnpm --filter design-system-demo --filter design-system-styles --parallel --stream start",
-    "styles:start": "pnpm --filter design-system-styles storybook:serve",
-    "components:start": "pnpm --filter design-system-components start",
-    "intranet-header:start": "pnpm --filter design-system-components-angular start",
+    "demo": "pnpm --filter design-system-demo --filter design-system-styles --parallel --stream start",
+    "styles": "pnpm --filter design-system-styles storybook:serve",
+    "components": "pnpm --filter design-system-components start",
+    "intranet-header": "pnpm --filter design-system-components-angular start",
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply package.json]: Done, but now I have to type a lot

File: .github/workflows/release.yaml [hunk 6]
```diff
@@ -51,3 +51,9 @@ jobs:
         run: |
           netlify link --id $NETLIFY_SITE_ID
           netlify deploy --build false --dir packages/demo/dist/demo --prod
+
+      # Publish storybook
+      - name: Publish storybooks
+        if: steps.changesets.outputs.published == 'true'
+        id: storybook
+        run: pnpm --filter design-system-documentation run chromatic --project-token=${{ secrets.STORYBOOK_TOKEN }}
```
Comment (oliverschuerch) [inline .github/workflows/release.yaml:59]:
```suggestion
        run: pnpm --filter design-system-documentation run chromatic --project-token=${{ secrets.CHROMATIC_STORYBOOK_TOKEN }}
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**